_includes/pbc/all/install-features/202311.0/install-the-agent-assist-feature.md

```diff
@@ -312,7 +312,7 @@ Enable the following behaviors by registering the plugins:
 
 | PLUGIN                                                  | DESCRIPTION                                                                                               | PREREQUISITES | NAMESPACE                                                            |
 |---------------------------------------------------------|-----------------------------------------------------------------------------------------------------------|---------------|----------------------------------------------------------------------|
-| AgentPageSecurityPlugin                                 | Registers security firewalls, access rules, impersonate rules, login and logout handlers for agent users. | None          | SprykerShop\Yves\AgentPage\Plugin\Security                           |
+| YvesAgentPageSecurityPlugin                             | Registers security firewalls, access rules, impersonate rules, login and logout handlers for agent users. | None          | SprykerShop\Yves\AgentPage\Plugin\Security                           |
```

Comment:
Symfony Security release is not part of `202311.0` release, so, we need to create documents for the new release and adjust them. And keep `202311.0` untouched.

Reply:
Do we know the date of the next release and what kind of directory we have to create instead of `202311.0`?

cc: @andriitserkovnyi